OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -72,6 +72,25 @@ Note: you can also mount the file directly as follows.
 
 Once the mounting is done, the custom_mod_wl_apache.conf will replace the built-in version of the file.
 
+## Enable SSL Access From User to Apache HTTP Server
+### Start Apache HTTP Server in a Container with SSL Support
+You can enable SSL support from user to Apache HTTP server when start **Apache HTTP Server with PlugIn** in a container using the following docker run command.
```

Comment:
You can enable SSL support from user to Apache HTTP server when starting the **Apache HTTP Server with PlugIn** in a container using the following Docker run command.